templates/concert-ticket/pdf_concert_ticket.go

```diff
@@ -0,0 +1,145 @@
+/*
+ * This example demonstrates creating a concert ticket using the creator templates.
+ *
+ * Run as: go run pdf_concert_ticket.go
+ */
+
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"log"
+	"os"
+	"text/template"
+	"time"
+
+	"github.com/boombuler/barcode"
+	"github.com/boombuler/barcode/qr"
+	"github.com/unidoc/unipdf/v3/common"
+	"github.com/unidoc/unipdf/v3/common/license"
+	"github.com/unidoc/unipdf/v3/creator"
+	"github.com/unidoc/unipdf/v3/model"
+)
+
+// Field represents a single field with name and value.
+type Field struct {
+	FieldName  string `json:"field_name"`
+	FieldValue string `json:"field_value"`
+}
+
+// Ticket holds all data related to a ticket.
+type Ticket struct {
+	EventTime         string   `json:"event_time"`
+	TicketNumber      string   `json:"ticket_number"`
+	Detail            []Field  `json:"ticket_detail"`
+	RulesOfAttendance []string `json:"rules_of_attendance"`
+	RulesOfPurchase   []string `json:"rules_of_purchase"`
+}
+
+func init() {
+	// Make sure to load your metered License API key prior to using the library.
+	// If you need a key, you can sign up and create a free one at https://cloud.unidoc.io.
+	err := license.SetMeteredKey(os.Getenv(`UNIDOC_LICENSE_API_KEY`))
+	if err != nil {
+		panic(err)
+	}
+
+	common.SetLogger(common.NewConsoleLogger(common.LogLevelDebug))
+}
+func main() {
+	c := creator.New()
+	c.SetPageMargins(20, 20, 20, 20)
+	// Create qrCode.
+	qrCode, err := createQRCode("https://github.com/unidoc/unipdf-examples/tree/master/templates/concert-ticket/", 500, 500)
+	if err != nil {
+		panic(err)
+	}
+	// Read ticket data.
+	ticket, err := readTemplateData("./concert-ticket.json")
+	if err != nil {
+		panic(err)
+	}
+	// Read template file.
+	tpl, err := readTemplate("./templates/main.tpl")
+	if err != nil {
+		log.Fatal(err)
+	}
+	// Draw main content template.
```

Comment:
Can you add an empty line above each of the comments in the `main` function please?